build/x86_64_bionic/Dockerfile

```diff
@@ -27,6 +27,7 @@ RUN apt-get -y update \
         lsb-release \
         make \
         pkg-config \
-        python-dev \
+        python2-dev \
```

Comment:
Rats, I thought I tested them all, but I forgot bionic. 😦 It didn't have python2-dev, only python-dev. Sent #136.